### ISSUE TITLE
 Assertion Failed: You cannot use class as an attributeBinding, use classNameBindings instead.

### ISSUE THREAD

**Issue by EWhite613** [DockYard/ember-one-way-controls#80]
Getting the error https://travis-ci.org/ciena-frost/ember-frost-core/jobs/123647368#L2073 with all my integration tests that use checkbox when using > 0.5.4 releases. 

I switched to using 
```
{{one-way-checkbox
    id=inputId
    class='input'
    checked=isChecked
    disabled=disabled
    name=name
    update=(action 'input')
}}
```

From

```
{{one-way-checkbox
    type='checkbox'
    id=inputId
    class='input'
    checked=isChecked
    disabled=disabled
    name=name
    update=(action 'input')
}}
```

But I still get the error. The error comes from dynamic attribute binding found here: https://github.com/DockYard/ember-one-way-controls/blob/master/addon/-private/dynamic-attribute-bindings.js

**Comment by martndemus** [comment]
Thanks for reporting! Will add tests and fix!

**Comment by EWhite613** [comment]
Seems like it's just trying to bind the class attribute as an attribute binding. When its meant to be a css class

**Comment by martndemus** [comment]
Fixed in #81
